examples/small-coalton-programs/src/oldies.coal

```diff
@@ -0,0 +1,13 @@
+(package small-coalton-programs/oldies
+  "The venerable fact and fib functions"
```

Comment:
It seems to be a fib about `fib` (counter to the fact of `fact`).

Potential "serious" package name- `recursive-basics`.